default.yaml

```diff
@@ -1,26 +1,28 @@
 # This is a default registries.d configuration file.  You may
 # add to this file or create additional files in registries.d/.
 #
-# lookaside: indicates a location that is read and write
-# lookaside-staging: indicates a location that is only for write
+# lookaside: a location simple signing signatures are read from
```

Comment:
```suggestion
# lookaside: a location where simple signing signatures are read from
```
? Same below

Reply:
That change was actually wrong, we do write to lookaside: . So I rewrote it completely differently again.